src/main/java/at/hannibal2/skyhanni/features/foraging/ForagingTrackerLegacy.kt

```diff
@@ -159,7 +159,17 @@ object ForagingTrackerLegacy {
      */
     val enchantedBookPattern by patternGroup.pattern(
         "bonus-gift.enchanted-book",
-        "(?:§.)+Enchanted Book \\((?:§.)+(?<book>.*) (?<tier>[IVCLX])(?:§.)+\\)"
+        " *(?:§.)+Enchanted Book \\((?:§.)+(?<book>.*) (?<tier>[IVCLX])(?:§.)+\\)"
+    )
+
+    /**
+     * REGEX-TEST: §r§7A §r§dPhanpyre §r§7fell from the Tree!
+     * REGEX-TEST: §r§7A §r§dPhanflare §r§7fell from the Tree!
+     * REGEX-TEST: §r§7A §r§dDreadwing §r§7fell from the Tree!
+     */
+    val phantomSpawnPattern by patternGroup.pattern(
```

Comment:
if we do not use this regex pattern in the 1.8 version of the mod, we can move it into the 1.21 foraging tracker file